release-notes/opensearch-knn.release-notes-2.0.0.0.md

```diff
@@ -0,0 +1,17 @@
+## Version 2.0.0.0 Release Notes
+
+Compatible with OpenSearch 2.0.0
+
+### Enhancements
+
+* Add Abstraction for BWC tests with a basic test ([#388](https://github.com/opensearch-project/k-NN/pull/388))
+* Add restart upgrade BWC Tests ([#387](https://github.com/opensearch-project/k-NN/pull/387))
+
+### Bug Fixes
```

Comment:
Should we remove these headings if we don't have any PRs to add in that section?

Reply:
I had the same question, my reason to leave those in the notes - it's clean indication that we do not have any changes for that category. Does it make sense?

Reply:
Ok, checked previous [notes](https://github.com/opensearch-project/k-NN/blob/main/release-notes/opensearch-knn.release-notes-1.1.0.0.md), seems that in general rule is - no change no category. Let me remove empty headings

Reply:
I got your point. But, in all the previous release notes we are only adding the sections that contains PRs. Just want to make things look uniform.

Reply:
Sure, I agree. Just pushed commit that corrects this, please check